index.html

```diff
@@ -583,19 +583,30 @@ <h1>Gradient Border</h1>
             <div class="preview" id="gradient-border-color-preview"></div>
 
             <label class="radius" for="gradient-border-radius">
-              <input type="checkbox" id="gradient-border-radius" name="check" />
-              <span> Include Radius </span>
-              <label id="radius-input-label" for="gradient-border-input">
-                <input
-                  type="number"
-                  id="gradient-border-input"
-                  name="desired-radius"
-                  placeholder="Enter a number e.g. 50"
-                  class="toggle-radius"
-                  autofocus
-                  required
-                />
-              </label>
+              <div class="container-fix-1">
```

Comment:
look at how other elements have their classname and use better class names accordingly, technically this is telling you to use your sense + good observation skills.

You are just working on things without being attentive